sdb/command.py

```diff
@@ -84,3 +84,25 @@ def call(self,
              objs: Iterable[drgn.Object]) -> Optional[Iterable[drgn.Object]]:
         # pylint: disable=missing-docstring
         raise NotImplementedError
+
+    @classmethod
+    def help(cls, name: str, verbose: bool = False):
+        """
+        Print a help message for the command based on the documentation
+        string for the class. This assumes the documentation uses the form:
+        <one line description of command>
+        <optional, and possibly multi-line description of command>
+        :type name: bool
+        """
+        cmd_name = '{0}: '.format(name)
+        docstr = inspect.getdoc(cls)
```

Comment:
I don't think this will work correctly when run on a command that doesn't have a command specific docstring; e.g.
```
> help spa
spa: A Locator is a command that locates objects of a given type.
```
It looks like this "falls back" to the docstring of a parent class, which I don't think is what we want.

Reply:
True, but the goal is to have a docstring for each command

Reply:
Yea, that's true.. so I'd be OK "fixing" this by simply adding a docstring to all commands. We can close this issue based on that.